WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -9,19 +9,21 @@
 using WalletWasabi.Helpers;
 using WalletWasabi.Logging;
 using WalletWasabi.WabiSabi.Backend.Models;
+using WalletWasabi.WabiSabi.Backend.Rounds;
 
 namespace WalletWasabi.WabiSabi.Client
 {
 	public class AliceClient
 	{
-		public AliceClient(uint256 roundId, ArenaClient arenaClient, Coin coin, FeeRate feeRate, BitcoinSecret bitcoinSecret)
+		public AliceClient(uint256 roundId, ArenaClient arenaClient, Coin coin, FeeRate feeRate, BitcoinSecret bitcoinSecret, RoundStateUpdater roundStatusUpdater)
```

Comment:
The `RoundStateUpdater` is not required for the `AliceClient` class but for the `ConfirmConnectionAsync` method only so, I would pass it as a parameter to that method only.

Reply:
ACK